documentation/book/assembly-logging.adoc

```diff
@@ -22,4 +22,4 @@ include::proc-kafka-inline-logging.adoc[leveloffset=+1]
 
 include::proc-kafka-external-logging.adoc[leveloffset=+1]
 
-Garbage collector (GC) logging can also be enabled (or disabled). For more information on GC, see xref:ref-jvm-options-{context}[]
+Garbage collector (GC) logging can also be enabled (or disabled). For more information on GC, see xref:ref-jvm-options-deployment-configuration-kafka[]
```

Comment:
why "deployment-configuration-kafka" ? even because the JVM options are not only about kafka but even other resources?

Reply:
thanks for the explanation, so it's just a workaround not the final solution as far as I understood. I am going to approve but please take a note about fixing it in the future.

Reply:
I will do. The future plan is to convert all cross-references to links.